fava/core/charts.py

```diff
@@ -165,11 +165,16 @@ def net_worth(self, interval):
         txn = next(transactions, None)
         inventory = Inventory()
 
+        today = datetime.datetime.today().replace(
+            hour=0, minute=0, second=0, microsecond=0)
         for date in self.ledger.interval_ends(interval):
             while txn and txn.date < date:
                 for posting in filter(lambda p: p.account.startswith(types),
                                       txn.postings):
-                    inventory.add_position(posting)
+                    inventory.add_amount(
```

Comment:
Maybe a short comment here why we do the calculations in this slightly odd way would be good.

Reply:
Done.